packages/collector/test/apps/agentStub.js

```diff
@@ -139,10 +139,14 @@ app.post(
 app.post(
   '/com.instana.plugin.nodejs/traces.:pid',
   checkExistenceOfKnownPid(function handleTraces(req, res) {
+    // eslint-disable-next-line no-console
+    console.log('received spans in agent', req.body);
     if (rejectTraces) {
       return res.sendStatus(400);
     }
     if (!dropAllData) {
+      // eslint-disable-next-line no-console
+      console.log('push data to receive');
```

Comment:
~~Similar thing here, -1 on this.~~ nevermind, guess this will be removed later as well